src/main/java/uk/gov/hmcts/sptribs/common/event/page/CaseUploadDocuments.java

```diff
@@ -32,7 +32,7 @@ public void addTo(PageBuilder pageBuilder) {
                     + "\n- a maximum of 100MB in size (large files must be split)\n"
                     + "\n- labelled clearly, e.g. applicant-name-B1-form.pdf\n\n")
             .complex(CaseData::getCicCase)
-            .optionalWithLabel(CicCase::getApplicantDocumentsUploaded, "File Attachments")
+            .mandatoryWithLabel(CicCase::getApplicantDocumentsUploaded, "File Attachments")
```

Comment:
@gmmagruder - You can add some tests for this.

Reply:
Yeah can probably use any existing mid event tests for that page